### ISSUE TITLE
Feature request: Support Angular 7

### ISSUE THREAD

**Issue by altso** [FortAwesome/angular-fontawesome#106]
### Describe the problem you'd like to see solved or task you'd like to see made easier

@fortawesome/angular-fontawesome throws a warning when installed along with Angular 7

```
npm WARN @fortawesome/angular-fontawesome@0.2.1 requires a peer of @angular/common@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @fortawesome/angular-fontawesome@0.2.1 requires a peer of @angular/core@^6.0.0 but none is installed. You must install peer dependencies yourself.
```

### Why would other angular-fontawesome users care about this?

Because everyone will eventually switch to Angular 7

### On a scale of 1 (sometime in the future) to 10 (absolutely right now), how soon would you recommend we make this feature?

9 (tomorrow is fine)

### Feature request checklist

- [x] This is a single feature (i.e. not a re-write of all of Font Awesome)
- [x] The title starts with "Feature request: " and is followed by a clear feature name (Ex: `Feature request: moar cowbell`)
- [x] I have [searched for existing issues](https://github.com/FortAwesome/angular-fontawesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by altso** [comment]
I see it was resolved in 0.3.0

**Comment by robmadole** [comment]
@altso haha. Was just getting ready to ask ya :)
